src/Uno.UI.Composition/Composition/ShapeVisual.skia.cs

```diff
@@ -41,29 +46,19 @@ internal override void Render(in DrawingSession parentSession)
 		base.Render(in parentSession);
```

Comment:
```suggestion
		base.Render(in session);
```

It will have the same result as anyway the new `session` impacts only the `Canvas` that is state-full and shared between sessions, but I think it's better explain what we are doing.

Reply:
@dr1rrb Isn't `Opacity` affected as well? `session` has the opacity pushed, but `parentSession` doesn't. Then `base.Render` will call `BeginDrawing` which pushes the opacity again.

So I think if we used `session` we may push opacity twice? Will see what the CI has to say though.

Reply:
Yes, confirmed it has an impact on some opacity samples. I'll keep it `parentSession` for now.

Reply:
Indeed we are also pushing opacity ... what's weird here is that we are drawing children in the `parentSession` but we are applying the clipping through the `session` that we are keeping alive on the side :/ The best approach here would be to create a `childrenSession` that applies clipping but not opacity.

BTW That was the purpose of the `Render` vs `Draw` methods.